actors_tools/GlobalActorReader/GlobalActorReader.hh

```diff
@@ -165,6 +175,16 @@ private:
     Variable<TVector3, true> PolarizationAfterProcess;
     //Process name
     Variable<std::string, true> ProcessName;
+    //Parent ID
```

Comment:
I think in this case those comments are not necessary. They are redundant cause from the variable name you would learn the same

Reply:
Or you need it for some doxygen ?

Reply:
I will extend description of this variables - this for people who don't know what specific variable does and means. Moreover I think about creating doxygen documentation in nearest future.